include/boost/math/policies/error_handling.hpp

```diff
@@ -21,10 +21,8 @@
 #include <cstdint>
 #include <boost/math/policies/policy.hpp>
 #include <boost/math/tools/precision.hpp>
-#ifndef BOOST_NO_EXCEPTIONS
 #include <stdexcept>
 #include <boost/math/tools/throw_exception.hpp>
-#endif
```

Comment:
I'm not entirely sure this is correct because I believe some platforms that do not support exceptions likewise do not have the header `<stdexcept>`